process/smartContract/atArgumentParser_test.go

```diff
@@ -4,35 +4,34 @@ import (
 	"encoding/hex"
 	"testing"
 
-	"github.com/ElrondNetwork/elrond-go/process"
 	"github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestNewAtArgumentParser(t *testing.T) {
```

Comment:
Please move this file in elrond-vm-common

Reply:
We don't run tests in `elrond-vm-common` very often. Can we leave the file here, to ensure continuous testing?